Enable etnaviv support in i.MX8MQ EVK devicetree

Some further investigation is required into how to manage this upstream
in meta-freescale, as it is not required for the i.MX6 platforms AFAIK.

Upstream-Status: pending
Signed-off-by: Scott Murray <scott.murray@konsulko.com>
---

diff --git a/arch/arm64/boot/dts/freescale/imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/imx8mq-evk.dts
index c3f8a24c3943..754fbfe30f0a 100755
--- a/arch/arm64/boot/dts/freescale/imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mq-evk.dts
@@ -1224,7 +1224,7 @@ &vpu_v4l2 {
 	status = "okay";
 };
 
-&gpu3d {
+&gpu {
 	status = "okay";
 };
 
